src/wix/WixToolset.Core/CommandLine/VersionCommand.cs

```diff
@@ -14,15 +14,17 @@ internal class VersionCommand : ICommandLineCommand
 
         public bool StopParsing => true;
 
+        private IServiceProvider ServiceProvider { get; }
+
         public Task<int> ExecuteAsync(CancellationToken cancellationToken)
         {
             // $(GitBaseVersionMajor).$(GitBaseVersionMinor).$(GitBaseVersionPatch)$(GitSemVerDashLabel)+$(Commit)
-            Console.WriteLine("{0}.{1}.{2}{3}+{4}", ThisAssembly.Git.BaseVersion.Major
-                                                  , ThisAssembly.Git.BaseVersion.Minor
-                                                  , ThisAssembly.Git.BaseVersion.Patch
-                                                  , ThisAssembly.Git.SemVer.DashLabel
-                                                  , ThisAssembly.Git.Commit);
-
+            Console.WriteLine(
+                ThisAssembly.Git.BaseVersion.Major + "." +
+                ThisAssembly.Git.BaseVersion.Minor + "." +
+                ThisAssembly.Git.BaseVersion.Patch +
+                ThisAssembly.Git.SemVer.DashLabel + "+" +
+                ThisAssembly.Git.Commit);
```

Comment:
Why switch to a bunch of string concatenations instead of letting `Console.Writeline()` handle the single string format internally? The compiler _probably_ figures out all those string concatenations can be turned into a `String.Concat()` call but it doesn't seem worth it to test the compiler this way. It won't matter in this case but avoiding string allocations is generally a good idea. Is there some reason this is better?

Reply:
I just thought this was a better representation of the desired output. It's a personal choice (because I don't like 'magic'/'I don't know how this is implemented in the writeline case') so if you prefer that style I can easily revert.

Reply:
Yeah. I would recommend that.

Reply:
No problem, done